specs/swagger.yaml

```diff
@@ -176,7 +201,31 @@ paths:
               schema:
                 $ref: "#/components/schemas/PredictionWithTime"
         "400":
-          description: Session not active, or model not initialized 
+          description: Session not active, or model not initialized
+  /v1/session/{id}/prediction/output:
+    parameters:
+      - name: id
+        in: path 
+        required: true
+        schema:
+          type: integer
+      - name: return_format
+        description: Format for the returned state
+        in: query
+        required: false
+        schema:
+          $ref: "#/components/schemas/ReturnFormat"
+    get:
+      description: Get the predicted output at save points (defined by pred_cfg['save_pts'] or pred_cfg['save_freq'])
+      responses:
+        "200":
+          description: Okay response
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/PredictionWithTime"
```

Comment:
This name works, as long as the schema is the same for predicted state and predicted output...

Reply:
That's right. Prediction and StateReturn (now PredictionPoint) are the same schema regardless of if you're requesting States, outputs, event states, or performance metrics